apis/r/tests/testthat/test-shape.R

```diff
@@ -56,6 +59,60 @@ test_that("SOMADataFrame shape", {
       expect_true(is.na(sjid_maxshape))
     }
 
+    dom = sdf$domain()
+    mxd = sdf$maxdomain()
+
+    # First check names
+    if (case_name == "one") {
+      expect_equal(names(dom), c("soma_joinid"))
+      expect_equal(names(mxd), c("soma_joinid"))
+
+    } else if (case_name == "two") {
+      expect_equal(names(dom), c("soma_joinid", "foo"))
+      expect_equal(names(mxd), c("soma_joinid", "foo"))
+
+    } else if (case_name == "three") {
+      expect_equal(names(dom), c("soma_joinid", "baz"))
+      expect_equal(names(mxd), c("soma_joinid", "baz"))
+
+    } else if (case_name == "four") {
+      expect_equal(names(dom), c("baz", "foo"))
+      expect_equal(names(mxd), c("baz", "foo"))
+    }
```

Comment:
These test modifications can be simplified:
 - we're checking that the names of `dom` and `mxd` are the same as `index_column_names` for each case; rather than hard-coding the cases, we can use the already-defined variable to perform the check for us. This also allows us to add new cases should we desire
```suggestion
    # First check names
    expect_equal(names(dom), index_column_names)
    expect_equal(names(mxd), index_column_names)
```